fpm/src/fpm_command_line.f90

```diff
@@ -240,255 +287,372 @@ function is_fortran_name(line) result (lout)
-    '                   compiler optimization flags are used.                        ', &
-    '   --list          list candidates instead of building or running them          ', &
-    '   -- ARGS         Arguments to pass to executables/tests                       ', &
-    '   --help          Show this help text and exit. Valid for all subcommands.     ', &
-    '   --version       Show version information and exit. Valid for all subcommands.', &
-    'EXAMPLES                                                                        ', &
-    ' sample commands:                                                               ', &
-    '                                                                                ', &
-    '    fpm build                                                                   ', &
-    '    fpm test                                                                    ', &
-    '    fpm run                                                                     ', &
-    '    fpm new --help                                                              ', &
-    '    fpm new mypackage --with-executable --with-test                             ', &
-    '    fpm run myprogram --release -- -x 10 -y 20 --title "my title"               ', &
-    'SEE ALSO                                                                        ', &
-    ' The fpm(1) home page is https://github.com/fortran-lang/fpm                    ', &
+    'NAME                                                                   ', &
+    '   fpm(1) - A Fortran package manager and build system                 ', &
+    '                                                                       ', &
+    'SYNOPSIS                                                               ', &
+    '   fpm SUBCOMMAND [SUBCOMMAND_OPTIONS]                                 ', &
+    '                                                                       ', &
+    '   fpm --help|--version|--list                                         ', &
+    '                                                                       ', &
+    'DESCRIPTION                                                            ', &
+    '   fpm(1) is a package manager that helps you create Fortran projects  ', &
+    '   from source.                                                        ', &
+    '                                                                       ', &
+    '   Most significantly fpm(1) lets you draw upon other fpm(1) packages  ', &
+    '   in distributed git(1) repositories as if the packages were a basic  ', &
+    '   part of your default programming environment, as well as letting    ', &
+    '   you share your projects with others in a similar manner.            ', &
+    '                                                                       ', &
+    '   See the fpm(1) repository at https://fortran-lang.org/packages      ', &
```

Comment:
The package index currently does not guarantee fpm support, but the registry at https://fortran-lang.org/packages/fpm does.